src/scripts/modules/media/body/content-baked.less

```diff
@@ -302,14 +302,9 @@ h1.example-title .text {
   }
 
   img {
-    float: left;
-    margin-right: 10px;
-  }
-
-  ol {
-    overflow: hidden;
+    display: block;
+    padding-bottom: 1em;
```

Comment:
Use margin instead of padding, so if anyone adds background/border or anything to this element, it will look ok too.